src/UserAdmin/src/Smart.FA.Catalog.Web/Pages/Admin/Trainings/Create/Index.cshtml.cs

```diff
@@ -44,13 +40,13 @@ public async Task<IActionResult> OnPostAsync()
             return Page();
         }
 
-        var request = CreateTrainingViewModel.MapToRequest(UserIdentity.CurrentTrainer.Id, UserIdentity.CurrentTrainer.DefaultLanguage);
-        var reponse = await Mediator.Send(request);
+        var createTrainingRequest = CreateTrainingViewModel.MapToRequest(UserIdentity.CurrentTrainer.Id, UserIdentity.CurrentTrainer.DefaultLanguage);
+        var response = await Mediator.Send(createTrainingRequest);
```

Comment:
The var for the request has details in the name but the response does not.
We should choose 1 approach and be consistent with it.
```suggestion
        var request = CreateTrainingViewModel.MapToRequest(UserIdentity.CurrentTrainer.Id, UserIdentity.CurrentTrainer.DefaultLanguage);
        var response = await Mediator.Send(createTrainingRequest);
```

Reply:
That was updated and set a meaningful name.